File: C/37.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	int n,i;
	float op,sal;
	sal=0;
	op=0;
	printf("Esse programa determina o salvario de um vandedor com base na comissâo!\n\n");

	printf("\n\nSalario do vendedor: ");
	scanf("%f",&sal);
		
	printf("\n\nVendas realizadas no mês: ");
	scanf("%d",&n);
	
	for(i=1;i<=n;i++){
		printf("\n\nValor da venda %d: ",i);
		scanf("%f",&op);	
		if(op<1000){
			sal=sal+50;
		}
		if((op>1000)and(op<=2000)){
			sal=sal+70;
		}
		if(op>2000){
			sal=sal+110;
		}
	}
	printf("\n\nO salario no final do mês será de R$%.2f",sal);
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}

File: Python/37.py
sal=0
op=0
printf=print
printf("Esse programa determina o salvario de um vandedor com base na comissâo!\n\n")

sal = float(input("\n\nSalario do vendedor: "))

n = int(input("\n\nVendas realizadas no mês: "))
i=1
for i in range(1,n+1):
    op = float(input(f"\n\nValor da venda {i}: "))
    if(op<1000):
        sal=sal+50
    elif((op>1000)and(op<=2000)):
        sal=sal+70
    elif(op>2000):
        sal=sal+110

    i=i+1

printf(f"\n\nO salario no final do mês será de R${sal}")
printf("\n\nObrigado por usar nosso sistema!\n\n")
